components/KitchenSink.tsx

```diff
@@ -345,8 +345,8 @@ export const KitchenSink: React.FC = (): JSX.Element => {
                                     <Avatar.Text
                                         label="JS"
                                         size={40}
-                                        color={BLUIColors.blue[500]}
-                                        style={{ backgroundColor: BLUIColors.blue[50] }}
+                                        color={BLUIColors.blue[800]}
+                                        style={{ backgroundColor: BLUIColors.blue[100] }}
```

Comment:
were you able to use "primary container" and "on primary container" color from the theme?